web/src/engine/predictive-text/wordbreakers/src/main/default/index.ts

```diff
@@ -293,6 +293,24 @@ function isNonSpace(chunk: string, options?: DefaultWordBreakerOptions): boolean
  * @param text Text to find word boundaries in.
  */
 function findBoundaries(text: string, options?: DefaultWordBreakerOptions): number[] {
+  //#region Internal utility functions
```

Comment:
```suggestion
```

I feel like this is overkill -- regions should be used for large blocks of code, not internal functions. In Javascript, local functions are normal and we don't need to annotate them.